src/check-pkgs.js

```diff
@@ -71,13 +70,17 @@ async function checkPkgs(auth, host, namespace, config, log = console) {
   const packages = await getPackageList(auth, host, namespace, log);
 
   config.strains.forEach((strain) => {
-    if (strain.package && !(strain.package in packages)) {
-      log.error(`action package for the following strain: << ${strain.name} >> not deployed`);
+    const package = new RegExp(`(?<=${namespace}/).*`).exec(strain.package);
```

Comment:
this is not very efficient. you create the regexp for each strain, although the namespace never changes. also, using regexp looks a bit complicated...

also, what happens if the namespace in the `strain.package` doesn't match the namespace provided?

what about something along the lines of
```
const segments = strain.package.split('/');
if (segments.length > 1) {
  if (segments[0] !== namespace) throw error('packages with different namespaces are not suported');
  segments.shift();
}
const package = segments[0];

Reply:
Ok! I can do that! 

I'm confused about your second part. If the namespace in the strain.package doesn't match it'll be not Found, which seems like what should happen

Reply:
yes, but the error is different. if the package cannot be found, then you didn't deploy the package via `hlx deploy`. but if the namespace is wrong (and if we gonna support this), then there is a more fundamental problem, that you try to publish something and having a wrong wsk namespace setup.

however, I think this is all a bit wrong :-)

since every package that we `hlx deploy` will have a `hlx--static` action, eg:

```
$ hlxctl info /helix-pages/15c18adcdfe346441d5b2f422c1e44cfec10b71d
Actions in package /helix-pages/15c18adcdfe346441d5b2f422c1e44cfec10b71d
  cgi-bin-sitemap  nodejs:10
  hlx--static      sequence -> /helix-pages/helix-services/static@v1
  html             nodejs:10
  idx_json         nodejs:10
  plain_html       nodejs:10
```

it would be easier to check if that action exists, since it is anyways needed for successful operation.

so the better check you be to invoke ` https://adobeioruntime.net/api/v1/web/${namespace}/${package}/hlx--static/_status_check/healthcheck.json`. this tests both, if the package is present and the static action.

eg:
```
curl -sD- https://adobeioruntime.net/api/v1/web/helix-pages/15c18adcdfe346441d5b2f422c1e44cfec10b71d/hlx--static/_status_check/healthcheck.json
```

Reply:
^^ Thank you for this info!